src/main/java/Application.java

```diff
@@ -16,22 +16,23 @@
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.util.List;
+import org.apache.log4j.Logger;
 
 public class Application {
     private static Injector injector = Injector.getInstance("com.dev.cinema");
+    private static final Logger LOGGER = Logger.getLogger(Application.class);
 
     public static void main(String[] args) throws AuthenticationException {
         MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         Movie movie1 = new Movie();
         movie1.setTitle("Fast & Furious");
         movie1.setDescription("action");
         movieService.add(movie1);
-        movieService.getAll().forEach(System.out::println);
         Movie movie2 = new Movie();
         movie2.setTitle("Matrix");
         movie2.setDescription("Science fiction");
         movieService.add(movie2);
-        movieService.getAll().forEach(System.out::println);
+        movieService.getAll().forEach(movie -> LOGGER.info(movie.toString()));
```

Comment:
method reference(toString() will be called automatically

Reply:
done